src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-deactivate-pop-up/ubs-admin-tariffs-deactivate-pop-up.component.ts

```diff
@@ -83,6 +83,7 @@ export class UbsAdminTariffsDeactivatePopUpComponent implements OnInit, OnDestro
   }
 
   ngOnInit(): void {
+    this.getCouriers();
```

Comment:
I guess, that it's bad idea to run this method at the beginning of ngOnInit method. It method getCouriers we use variable currentLanguage, which is defined in 90 line. Therefore in method getCouriers we can get only english name as far as this.currentLanguage === 'ua' will be always false.